src/index.ts

```diff
@@ -74,6 +83,59 @@ class Main {
         ShapeUtils.addShapeDrawer(ShapeType.character, textDrawer);
         ShapeUtils.addShapeDrawer(ShapeType.image, new ImageDrawer());
     }
+
+    /**
+     * init method, used by imports
+     */
+    public init(): void {
+        if (typeof window !== "undefined" && window) {
+            /* particles.js compatibility */
+            const tsParticles = this;
+
+            /**
+             * Loads the provided options to create a [[Container]] object.
+             * @deprecated this method is obsolete, please use the new tsParticles.load
+             * @param tagId the particles container element id
+             * @param params the options object to initialize the [[Container]]
+             */
+            window.particlesJS = (tagId: string, params: RecursivePartial<IOptions>) => {
+                tsParticles.load(tagId, params);
+            };
+
+            /**
+             * Loads the provided json with a GET request. The content will be used to create a [[Container]] object.
+             * @deprecated this method is obsolete, please use the new tsParticles.loadJSON
+             * @param tagId the particles container element id
+             * @param pathConfigJson the json path to use in the GET request
+             * @param callback the function called after the [[Container]] object is loaded that will be passed as a parameter
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5308283)